charts/external-secrets/README.md

```diff
@@ -0,0 +1,234 @@
+# External Secrets Profile Installation Guide
+
+**Table Of Contents**
+- [External Secrets Profile Installation Guide](#external-secrets-profile-installation-guide)
+  - [Introduction](#introduction)
+  - [Components](#components)
+    - [Controller](#controller)
+    - [Custom Resource Definitions (CRDs)](#custom-resource-definitions-crds)
+      - [SecretStore](#secretstore)
+      - [ExternalSecret](#externalsecret)
+      - [Note](#note)
+    - [CRs](#crs)
+  - [Profile Components](#profile-components)
+  - [How to install with WGE on Kubernetes Cluster](#how-to-install-with-wge-on-kubernetes-cluster)
+  - [Notes on the creating the leaf cluster](#notes-on-the-creating-the-leaf-cluster)
+  - [Create Service Account on AWS](#create-service-account-on-aws)
+
+## Introduction
+
+External Secrets Operator is a Kubernetes operator that integrates external secret management systems like AWS Secrets Manager, HashiCorp Vault, Google Secrets Manager, Azure Key Vault and many more. The operator reads information from external APIs and automatically injects the values into a Kubernetes Secret.
+For more information refer to the docs [here](https://external-secrets.io/v0.6.1/)
+
+## Components
+
+The External Secrets operator consists of 3 main parts as the following
+
+### Controller
+
+The External Secrets Operator extends Kubernetes with Custom Resources, which define where secrets live and how to synchronize them. The controller fetches secrets from an external API and creates Kubernetes secrets. If the secret from the external API changes, the controller will reconcile the state in the cluster and update the secrets accordingly.
+
+### Custom Resource Definitions (CRDs)
+
+The operator needs 2 predefined CRDs to be installed on the cluster to be able to operate.
+
+#### SecretStore
+
+The idea behind the SecretStore resource is to separate concerns of authentication/access and the actual Secret and configuration needed for workloads
+
+#### ExternalSecret
+
+An ExternalSecret declares what data to fetch. It has a reference to a SecretStore which knows how to access that data. The controller uses that ExternalSecret as a blueprint to create secrets.
+
+#### Note
+
+The previous CRDs are namespaced, There is also a global, cluster-wide SecretStore that can be referenced from all namespaces. You can use it to provide a central gateway to your secret provider which are (ClusterSecretStore, ClusterExternalSecret)
+
+### CRs
+
+In order to use the operator you will need to define the SecretStore and the ExternalSecret(s) you will use. You can define one or more and add them to a GitRepository. Then a Kustomization will reference them to be installed on the cluster
+
+## Profile Components 
+
+1- [The HelmChart for External Secrets Operator](Chart.yaml)
+
+2- [Kustomization Reference to the CRs](templates/secrets.yaml)
```

Comment:
it could make sense to be more concrete to help the user with understanding 

`Kustomization Reference to the CRs` to `Kustomization Reference to secret stores CRs`

Reply:
mentioned also in https://github.com/weaveworks/profiles-catalog/pull/202#discussion_r1048339361 

that could be better to have `templates/secret-stores.yaml` instead of `templates/secrets.yaml` cause is less ambiguous